CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 - [\#952](https://github.com/Manta-Network/Manta/pull/952) update genesis [MA]
 
 ### Changed
+- [\#981](https://github.com/Manta-Network/Manta/pull/981) Ledger Error Handling [MACADO]
```

Comment:
Changelog is added to every release with github action. No need to edit manually 

Just need to add label to PR